scripts/build/build_appdeps.sh

```diff
@@ -22,7 +22,7 @@ if [ "$OSNAME" == "osx" ]; then
 elif [ "$OSNAME" == "ubuntu" ]; then
     RID=ubuntu.14.04-x64
 elif [ "$OSNAME" == "centos" ]; then
-    RID=centos.7.1-x64
+    RID=rhel.7.1-x64
```

Comment:
Looks like there is no CoreRT toolchain for centos or rhel.  @schellap - is this a known TODO?